gridcomps/Cap/FlapCLI.F90

```diff
@@ -5,47 +5,55 @@ module MAPL_FlapCLIMod
    use MPI
    use ESMF
    use FLAP
-   use MAPL_KeywordEnforcerMod
-   use MAPL_ExceptionHandling
+   use mapl_KeywordEnforcerMod
+   use mapl_ExceptionHandling
+   use mapl_CapOptionsMod, only: MAPL_FlapCli => MAPL_CapOptions
```

Comment:
Seems like the use of the name `MAPL_FlapCli` to rename the `MAPL_CapOptions` interface for the `new_CapOptions` procedure and still using it in this module as the name of the interface holding the `new_CapOptions_from_flap` procedure is causing conflicts. Commenting this line though makes it fail in the external components like Fvdycore.